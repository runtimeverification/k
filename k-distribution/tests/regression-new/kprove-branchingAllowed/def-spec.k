module DEF-SPEC
    imports DEF

    claim <k> a => d </k>

endmodule
