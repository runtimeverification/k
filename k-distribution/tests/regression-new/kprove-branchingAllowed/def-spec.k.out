  <k>
    b ~> .K
  </k>
#Or
  <k>
    c ~> .K
  </k>
